src/storage/StorageServer.cpp

```diff
@@ -96,10 +97,31 @@ bool StorageServer::initWebService() {
     return status.ok();
 }
 
+static std::shared_ptr<apache::thrift::concurrency::ThreadManager> newThreadManager() {
+    std::vector<size_t> threadNums;
+    folly::split(":", FLAGS_num_threads_per_priority, threadNums, false);
+    if (threadNums.size() != apache::thrift::concurrency::N_PRIORITIES) {
+        return nullptr;
+    }
+    return apache::thrift::concurrency::PriorityThreadManager::newPriorityThreadManager(
+            {{
+                threadNums[apache::thrift::concurrency::HIGH_IMPORTANT],
```

Comment:
HIGH_IMPORTANT is in usage?

Reply:
Hi, not in use yet, I'm not quit sure if the _priority_ declaration in storage.thrift is reasonable, do you think 0:2:32:32:2 is OK?